catalog/CAS/CAS_IGSNRR_PML_V2_v018.jsonnet

```diff
@@ -0,0 +1,205 @@
+local id = 'CAS/IGSNRR/PML/V2_v018';
+local versions = import 'versions.libsonnet';
+local version_table = import 'templates/IGSNRR_PML_versions.libsonnet';
+
+local subdir = 'CAS';
+
+local ee_const = import 'earthengine_const.libsonnet';
+local ee = import 'earthengine.libsonnet';
+local spdx = import 'spdx.libsonnet';
+local units = import 'units.libsonnet';
+local version_config = versions(subdir, version_table, id);
```

Comment:
This won't work until your new dataset is mentioned in the versions table, but we can't do that until the dataset is activated. Sorry for a difficult process, but for now you shouldn't import versions into this file - we will do it later when we activate the dataset.

Reply:
If I shouldn't import versions into this file, does that mean I should delete any scripts here related to versions?

Reply:
Right, remove any references to version_config. You can still have a local hartcoded 'version' variable.